### ISSUE TITLE
Error: passphrase must be specified or set as NSPW environment variable.

### ISSUE THREAD

**Issue by philipdenys** [ExploreConsulting/netsuite-sync#9]
The CLI never asks for a password, do we need to use the (export NSPW=) param?

**Comment by ShawnTalbert** [comment]
Yes though these days I would probably recommend using the Neetsuite SDF
cli to do what this utility does. Supposedly they are about to release a
nodejs version of the SDF cli so it will be similar?
